### ISSUE TITLE
Is there any way to replicate forms?

### ISSUE THREAD

**Issue by jivy59** [formtools/core#209]
I have many forms which are exactly the same except for the form and redirect urls.  Everything else, including the emails, etc. is exactly the same.

I do not want to replicate the data in the form, just the form itself so that all I need to do is update the form and redirect URLs when I add a new landing page.

**Comment by jivy59** [comment]
That does help.  I believe the backup feature will do the job for now. But
the other path sounds definitely worth exploring.

Thank you for the advice and the wonderful tool!

**Comment by jivy59** [comment]
After digging into your second suggestion, I realized what a beauty it was.  All of my websites are in sub-directories inside of a common directory.  I set the url of the "master" form to the common directory then added some code to automatically populate the redirect field to the appropriate sub-directories.  All of the data will be in the same database table, but that does not matter because all I am doing is sending the data to a separate CRM.  So far it is working like a charm and now I can create a new website in minutes instead of a full day.  Thanks for your help and suggestion
